module/VuFind/src/VuFind/I18n/Translator/TranslatorAwareTrait.php

```diff
@@ -101,6 +101,12 @@ public function getTranslatorLocale($default = 'en')
      */
     public function translate($target, $tokens = [], $default = null)
     {
+        if ($this->getTranslatorLocale() == "keys") {
+            return "*$target"
+                . (!empty($tokens) ? " | ['" . implode("', '", array_keys($tokens)) . "']" : "")
```

Comment:
This looks like it's only going to show the array keys, but seeing the values is useful too. Maybe it's worth something a little more complicated, like:

```
$callback = function ($key, $val) {
    return "$key = $val";
};
$tokenDetails = empty($tokens)
    ? ''
    : '| [' . implode(', ', array_map($callback, array_keys($tokens), array_values($tokens))) . ']';
return "*$target$tokenDetails*";
```

Reply:
Thanks! I'll try this out in more detail when I return from vacation next week, but it's looking good to me.